apps/prairielearn/python/prairielearn/grading_utils.py

```diff
@@ -20,22 +20,35 @@
 
 # This is a deprecated alias that will be removed in the future -- use the lowercase version instead.
 def is_correct_ndarray2D_dd(*args: Any, **kwargs: Any) -> bool:  # noqa: N802
+    """Check if a submitted 2D numpy array is correct within a certain number of decimal digits after the decimal place.
```

Comment:
Seems unnecessary to document a deprecated function. Should just include the deprecation statement, if anything.

Reply:
My thought is that since the docstring shows up on the user docs, we want a mention here.

Reply:
I disagree, if this is deprecated it should only direct people to use the non-deprecated version, and it should not appear in the user docs. This is to deter people from using this as much as possible.

Reply:
Good point, that is a much better idea. I'll see how to exclude it instead.

Reply:
Since functions without docstrings aren't included on the docs, I removed the docstrings from these.